src/Tuura/Concept/Circuit/Derived.hs

```diff
@@ -199,3 +201,14 @@ internals ints = interfaceConcept $ \s ->
                  if s `elem` ints
                  then Internal
                  else Unused
+
+combinationalGate :: Eq a => Expr a -> a -> CircuitConcept a
```

Comment:
I like this idea. The name of the function is becoming long, but it is good. We could do something like `complexGate` for just a `set` function, and `complexSRGate` for `set` and `reset`? Less understandable, I'll admit.